<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceUpgradeWidget</class>
 <widget class="QWidget" name="DeviceUpgradeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="ImportContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="ImportLabel">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(255,255,255,104);</string>
          </property>
          <property name="text">
           <string>import upgrade package</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="ImportSubContainer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>72</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>72</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="ImportSelectContainer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>72</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="ImportNoticeIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ImportNoticeLabel">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255,255,255,104);</string>
            </property>
            <property name="text">
             <string>import upgrade package</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>382</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="ImportBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
             </font>
            </property>
            <property name="text">
             <string>import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ImportingProgressContainer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>72</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="FileIconLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="ImportingContainer" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="PackageNameContainer" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>14</number>
                </property>
                <item>
                 <widget class="QLabel" name="FileNameLabel">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>file name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="InfoIcon">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="ProgressContainer" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>14</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="warningIcon_2">
                  <property name="minimumSize">
                   <size>
                    <width>27</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>27</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="res.qrc">:/res/images/ic_device_warning.svg</pixmap>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="InfoLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QProgressBar" name="ImportingProgressBar">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>4</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>4</height>
                   </size>
                  </property>
                  <property name="value">
                   <number>24</number>
                  </property>
                  <property name="textVisible">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="ImportingProgressLabel">
                  <property name="text">
                   <string>0%</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReselectPkgBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>reselect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CancelImportingBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="DevLstContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="DevLstHead" native="true">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,3,2,2">
         <item>
          <widget class="QLabel" name="DevLstNameTitle">
           <property name="styleSheet">
            <string notr="true">color: rgba(255,255,255,104);</string>
           </property>
           <property name="text">
            <string>    name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="DevLstSNTitle">
           <property name="styleSheet">
            <string notr="true">color: rgba(255,255,255,104);</string>
           </property>
           <property name="text">
            <string>sn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="DevLstVersionTitle">
           <property name="styleSheet">
            <string notr="true">color: rgba(255,255,255,104);</string>
           </property>
           <property name="text">
            <string>version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="DevLstStatusTitle">
           <property name="styleSheet">
            <string notr="true">color: rgba(255,255,255,104);</string>
           </property>
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DeviceListView" name="DevLstView">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="warningIcon">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/res/images/ic_device_warning.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="UpgradeNoticeLabel">
       <property name="styleSheet">
        <string notr="true">color: #FF2F6F;</string>
       </property>
       <property name="text">
        <string>upgrade may take long time, 5G wifi / cabel recommended</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="UpgradeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>102</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>102</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>upgrade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopUpgradeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>102</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>102</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>stop upgrade</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceListView</class>
   <extends>QListView</extends>
   <header>devicelistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
